roles/openshift_logging_elasticsearch/vars/default_images.yml

```diff
@@ -1,5 +1,5 @@
 ---
 __openshift_logging_elasticsearch_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') }}"
 __openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default('latest') }}"
-__openshift_logging_elasticsearch_proxy_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') | regex_replace('^(.*)/origin-$', '\\1/') }}"
+__openshift_logging_elasticsearch_proxy_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/') }}"
```

Comment:
this will not work, you set `openshift_logging_image_prefix` as a fact here:
https://github.com/ewolinetz/openshift-ansible/blob/6575595891f71e7d9b9ac929b9ba58117a65e17b/roles/openshift_logging/tasks/main.yaml#L16-L17

therefore it is never null and usage of `default()` makes no sense here

Reply:
This is so that if the role is used standalone we will still have a default.

Reply:
@wozniakjan ah ok, I see what you're saying here... so if we remove the fact setting in `openshift_logging` then we can properly fall back on the defaults for the roles. I was missing that point before...